internal/jsondb/db.go

```diff
@@ -79,6 +79,14 @@ func (db *JSONDatabase) List() ([]string, error) {
 	return names, nil
 }
 
+// Delete will delete the file from the database
+func (db *JSONDatabase) Delete(name string) error {
+	if len(name) == 0 {
+		return fmt.Errorf("missing jsondb document name")
+	}
+	return os.Remove(filepath.Join(db.dir, name+".json"))
```

Comment:
This is vulnerable to path traversal, however, I only see UUIDs passed in. So I am assuming we do not pass anything else in here. We are good.

Reply:
Right, and with it appending .json it's not terribly useful.